### PR TITLE
Issue #10083: Update inputs for AnnotationOnSameLineCheckTest

### LINKED ISSUE #10083: Update inputs for AnnotationOnSameLineCheckTest
Child of #10080

### PR DESCRIPTION
**Pull request by shashwatj07**
Resolves #10083

### CODE REVIEW

**Review by timurt — APPROVED** [review]

File: config/checkstyle_non_main_files_suppressions.xml [hunk 1]
```diff
@@ -71,6 +71,16 @@
...
 
+  <!-- config attributes cannot be wraped -->
+  <suppress id="lineLength"
+         files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck.java"/>
+  <suppress id="lineLength"
+         files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
+  <suppress id="lineLength"
+         files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
+  <suppress id="lineLength"
+         files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheckOnDifferentTokens.java"/>
```
Comment (romani) [inline config/checkstyle_non_main_files_suppressions.xml:82]: https://checkstyle.org/config_filters.html#SuppressionFilter

use lines, `lines="2-5"/>` and let presume token property will be always one of top

or even better is update:
https://github.com/checkstyle/checkstyle/blob/13481f2c410e4944ecf5ab93ec49948a523a0c82/config/checkstyle_checks.xml#L214

to skip lines that start with "tokens="

**Review by romani — CHANGES_REQUESTED** [review]

File: config/checkstyle_non_main_files_suppressions.xml (hunk 1 above)
Comment (shashwatj07) [reply config/checkstyle_non_main_files_suppressions.xml]: It was actually here:
https://github.com/checkstyle/checkstyle/blob/13481f2c410e4944ecf5ab93ec49948a523a0c82/config/checkstyle_non_main_files_checks.xml#L18-L24

Comment (shashwatj07) [reply config/checkstyle_non_main_files_suppressions.xml]: done now

File: config/checkstyle_non_main_files_checks.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
     <property name="id" value="lineLength"/>
     <!-- catch lines above 100 symbols -->
     <property name="format"
-             value="^(?!(.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd| \* \{@code| \* com\.)).{101,}$"/>
+             value="^(?!(.*href=|.*http(s)?:|tokens|import |(.* )?package |.* files=|.*\.dtd| \* \{@code| \* com\.)).{101,}$"/>
     <property name="message" value="Line should not be longer than 100 symbols"/>
   </module>
```
Comment (shashwatj07) [inline config/checkstyle_non_main_files_checks.xml:24]: We may want to change this to LineLengthCheck

**Review by romani — APPROVED** [review]
Ok to merge

File: config/checkstyle_non_main_files_checks.xml (hunk 2 above)
Comment (shashwatj07) [reply config/checkstyle_non_main_files_checks.xml:24]: @romani

**Pull request merged**